lib/js/components/Drawer/DrawerHeader/DrawerHeader.vue

```diff
@@ -0,0 +1,190 @@
+<template>
+	<div class="drawerHeader">
+		<div class="drawerHeader__titleWrapper">
+			<ds-button
+				v-if="isSecondLevel"
+				:icon-left="ICONS.FA_CHEVRON_LEFT"
+				:type="BUTTON_TYPES.TEXT"
+				class="drawerHeader__secondLevel"
+			>
+				Wróć
+			</ds-button>
+			<div :class="{ '-hidden': isSecondLevel }" class="drawerHeader__firstLevel">
+				<span
+					v-if="eyebrowText"
+					:class="{
+						'-isInteractive': isInteractiveEyebrow,
+						'-ellipsis': eyebrowEllipsis,
+					}"
+					class="drawerHeader__eyebrow"
+					@click="isInteractiveEyebrow && $emit('eyebrowClicked')"
+				>
+					{{ eyebrowText }}
+				</span>
+				<div class="drawerHeader__title">
+					<icon
+						v-if="leftIcon"
+						:icon="leftIcon"
+						:size="ICON_SIZES.X_SMALL"
+						class="drawerHeader__leftIcon"
+					/>
+					<span v-if="title" :class="{ '-ellipsis': titleEllipsis }">{{ title }}</span>
+					<pill v-if="pillLabel" :label="pillLabel" />
+				</div>
+			</div>
+			<icon-button
+				v-if="isClosable"
```

Comment:
is something wrong with size, based on figma it should be 32x32px now is 40x40px - so the whole component has incorrect height

Reply:
Probably we need:
```suggestion
				v-if="isClosable"
				:touchable="false"
```